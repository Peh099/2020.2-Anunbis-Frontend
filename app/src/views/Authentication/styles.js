import styled from 'styled-components';

export const AuthenticationStyle = styled.div`
  height: min(450px, 100%);
  width: min(400px, 35vw);
  display: flex;
  align-items: center;
  flex-direction: column;
`;

export const Content = styled.div`
  width: min(40vw, 100%);
  height: min(50vh, 100%);
  display: flex;
  justify-content: center;
  align-self: center;
  flex-direction: column;
  padding: 0px 0px 10px 0px;
`;

export const MenuOptions = styled.div`
  width: 100%;
  display: flex;
  padding: 20px;
  align-items: center;
  justify-content: space-evenly;
  flex-wrap: wrap;
`;

export const Option = styled.div`
  @media (max-height: 640px) {
    font-size: 11px;
  }
  font-size: 14px;
  text-decoration: ${(props) => (props.selected ? 'underline' : 'none')};
  cursor: pointer;
`;

export const Title = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 20px;
  font-weight: bold;
  margin-bottom: 20px;
  @media (max-height: 640px) {
    font-size: 14px;
  }
  @media (max-height: 100px) {
    display: none;
  }
`;
