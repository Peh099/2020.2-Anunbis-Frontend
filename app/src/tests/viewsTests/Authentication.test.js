import renderer from 'react-test-renderer';
import Authentication from '../../views/Authentication';

describe('Snapshot Authentication view', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<Authentication/>).toJSON() 
    expect(tree).toMatchSnapshot()
    })
})