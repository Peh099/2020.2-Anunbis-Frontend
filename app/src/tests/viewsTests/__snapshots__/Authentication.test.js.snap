// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Authentication view matches the snapshot 1`] = `
<div
  className="sc-bdnxRM ivnDX"
>
  <div
    className="sc-dlnjwi kpNdnw"
  >
    <div
      className="sc-hKFxyN kKsEfH btnLogin"
      onClick={[Function]}
      selected={true}
    >
      Login
    </div>
    <div
      className="sc-hKFxyN eJfuaj btnCadastro"
      onClick={[Function]}
      selected={false}
    >
      Cadastro
    </div>
  </div>
  <div
    className="sc-gtsrHT eHfyej"
  >
    <form
      className="sc-fujyAs iUoQcC"
      onSubmit={[Function]}
    >
      <div
        className="sc-gKAaRy btmTHI"
      >
        <div />
        <div
          className="sc-iCoGMd dBpElD error"
        />
      </div>
      <div
        className="sc-gKAaRy btmTHI"
      >
        <input
          className="sc-jrsJWt QesSX"
          name="email"
          placeholder="Email Instuticional"
          type="text"
        />
         
        <div
          className="sc-iCoGMd dBpElD error"
        />
      </div>
      <div
        className="sc-gKAaRy btmTHI"
      >
        <input
          className="sc-jrsJWt QesSX"
          name="password"
          placeholder="Senha"
          type="password"
        />
         
        <div
          className="sc-iCoGMd dBpElD error"
        />
      </div>
      <div
        className="sc-pNWdM kXmnwj"
      >
        <button
          className="sc-jSFjdj jthJeV"
        >
          CONFIRMAR
        </button>
      </div>
    </form>
  </div>
</div>
`;
